pkg/kubelet/kubelet_getters.go

```diff
@@ -466,3 +466,13 @@ func (kl *Kubelet) setCachedMachineInfo(info *cadvisorapiv1.MachineInfo) {
 	defer kl.machineInfoLock.Unlock()
 	kl.machineInfo = info
 }
+
+// getLastStableNodeAddresses returns the last observed node addresses.
+func (kl *Kubelet) getLastObservedNodeAddresses() []v1.NodeAddress {
+	node, err := kl.GetNode()
+	if err != nil || node == nil {
+		klog.ErrorS(err, "fail to obtain node from local cache", "node", kl.nodeName)
```

Comment:
I'm on the fence about logging an error here... on init while the node is getting its client cert and registering the node, this will be nil

Reply:
did I copied this from some place?

Reply:
let me add verbosity so it can be toggled

Reply:
updated to V(4)